Userland/Services/LookupServer/LookupServer.cpp

```diff
@@ -135,7 +142,17 @@ Vector<DNSAnswer> LookupServer::lookup(const DNSName& name, DNSRecordType record
         answers.append(answer_with_original_case);
     };
 
-    // First, try local data.
+    // First, try the hostname.
+    static DNSName s_hostname = get_hostname();
+    if (record_type == DNSRecordType::A && s_hostname == name) {
```

Comment:
Shouldn't we prefer to use `/etc/hosts` and the other mechanisms? This would match what other systems do.

Reply:
Yes, you're right.